### ISSUE TITLE
Badger 2040 documentation confusing/incorrect about image formatting between badge and images.

### ISSUE THREAD

**Issue by ConstantineK** [pimoroni/pimoroni-pico#718]
In the README for the badger2040 - https://github.com/pimoroni/pimoroni-pico/tree/main/micropython/examples/badger2040/README.md it recommends using the --binary and --resize flags for both the badge and your displayed images. 
The file extension for images in the unit are all .jpeg which is confusing (instead of bin which is the output of this program.) 
In addition, the badge picture cannot be resized to the larger size if you want to display it, --resize doesn't seem appropriate here. 

https://learn.pimoroni.com/article/getting-started-with-badger-2040 also mentions using badge.py to reference your badge image by placing it in the root of the directory with a specific name, but nowhere in the current code do I see that needing to be the case.

**Comment by ConstantineK** [comment]
After reviewing the code more appropriate advice would be something like:

```
# On WSL:
sudo apt install imagemagick-6.q16

# Then run the following on your already resized 104x128 image
convert badge.jpg  -flatten -monochrome +compress badge_bit.jpeg
```
Upload the image with Thonny (you may have to select and open the folder in the UI within the badger itself if you want it to prompt you to upload it in the subfolder you are targeting)

Then update your badges/badge.txt file and change line 7 to the path of your image.

Reboot/reset the badger and your image will be presentable.

**Comment by Gadgetoid** [comment]
Converting images is redundant since we added jpeg support. And Badger now lives here- https://github.com/pimoroni/badger2040

These old docs and tools probably need clearly deprecating sooner rather than later, but I had left them while the new Badger beds in.

**Comment by helgibbons** [comment]
Hopefully the situation is a bit less confusing now - Badger 2040 (and Badger 2040 W) now have their [own repo](https://github.com/pimoroni/badger2040) with their own docs and I've updated the [Learn article](https://learn.pimoroni.com/article/getting-started-with-badger-2040) to describe the new way of doing things. The links on both shop pages are also now updated to point to the new repo.

Closing this one for now, but do feel free to [open a new issue over on the Badger 2040 repo](https://github.com/pimoroni/badger2040/issues) if there are documentation tweaks you'd still like us to make.
